packages/custom-elements-languageserver/package.yaml

```
@@ -1,24 +1,15 @@
 ---
 name: custom-elements-languageserver
 description: |
-  The best way to utilize the Custom Elements Language Server is to enable the Custom Elements Manifest (CEM) in your
-  project by installing a CEM generator like one provided by The Open WC Team.
-
-  Generating a CEM in watch mode will provide you with the best user experience. If your dependencies ship with a Custom
-  Elements Manifest, those will be utilized also.
+  Custom Elements Language Server provides useful language features for Web Components. Features include code actions, completons, diagnostics and more.
 homepage: https://github.com/Matsuuu/custom-elements-language-server
 licenses:
   - BSD-3-Clause
```

The `languages` field is a required one so if there are no associated languages we'll need to:

```suggestion
  - BSD-3-Clause
languages: []
```